packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -175,16 +176,15 @@ export default function Sidebar({
     Features.FeedFilterModalOnboarding,
     flags,
   );
+
   const [isFirstSession, setIsFirstSession, isSessionLoaded] =
-    usePersistentContext(FIRST_TIME_SESSION, true);
+    usePersistentContext(FIRST_TIME_SESSION, isFirstVisit);
```

Comment:
One edge case that comes into my mind, is the user enters daily.dev for the first time, sees the onboarding but does not finish it. Comes back again and suddenly they will not see it any more right?

Reply:
Sorry, my assumption was this was how it was supposed to work.
So we want to keep showing this until they act?

It might become annoying for users if they have to close this modal time over?

Reply:
If they dismissed it, it should be gone forever I agree. I originally meant in cases where they didn't interact at all or didn't finish the process we want to keep showing it until they dismiss manually. Does that make sense?